crates/swc_ecma_transforms_compat/src/es2017/async_to_generator.rs

```diff
@@ -444,23 +447,18 @@ impl Fold for Actual {
                 return_type: Default::default(),
                 type_params: Default::default(),
             },
-            ..prop
-        }
+            key: prop.key.take(),
+            ..*prop
+        };
     }
 
-    fn fold_prop(&mut self, n: Prop) -> Prop {
+    fn visit_mut_prop(&mut self, n: &mut Prop) {
         let old = self.in_object_prop;
         self.in_object_prop = true;
 
-        let n = n.fold_children_with(self);
+        n.visit_mut_children_with(self);
 
         self.in_object_prop = old;
-
-        n
-    }
-
-    fn fold_stmts(&mut self, n: Vec<Stmt>) -> Vec<Stmt> {
-        n
     }
```

Comment:
Seems like `fold_stmts` is removed?

Reply:
yes. doesn't seem to do anything? if it's visit_mut, should it explicitly noop?